.env.template

```diff
@@ -4,3 +4,9 @@ FLASK_ENV=development
 
 # Change the following values for local development.
 SECRET_KEY=secret-key
+
+API_KEY=
+API_TOKEN=
+TRELLO_BOARD_ID=
+TRELLO_TODO_LIST_ID=
+
```

Comment:
Note that you haven't added the `TRELLO_DONE_LIST_ID` here which meant I initially missed that; it'd be good to make sure this is comprehensive